specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2020-10-20/healthbot.json

```diff
@@ -0,0 +1,630 @@
+            "schema": {
+              "$ref": "#/definitions/BotResponseList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.HealthBot/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "Lists all the available HealthBot operations.",
+        "operationId": "Operations_List",
+        "x-ms-examples": {
+          "Get Operations": {
+            "$ref": "./examples/GetOperations.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/AvailableOperations"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CheckNameAvailabilityRequestBody": {
+      "description": "The request body for a request to to check availability of a bot name.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "the name of the bot for which availability needs to be checked.",
+          "type": "string"
+        },
+        "type": {
+          "description": "the type of the bot for which availability needs to be checked",
+          "type": "string"
+        }
+      }
+    },
+    "CheckNameAvailabilityResponseBody": {
+      "description": "The response body returned for a request to check availability of a bot name.",
+      "type": "object",
+      "properties": {
+        "nameAvailable": {
+          "description": "indicates if the bot name is valid and available.",
+          "type": "boolean"
+        },
+        "reason": {
+          "description": "Reason for the result",
+          "type": "string"
+        },
+        "message": {
+          "description": "additional message from server",
+          "type": "string"
+        }
+      }
+    },
+    "Resource": {
```

Comment:
This is missing tags (which should be patchable as well).  You'll save yourself some work if you reference the common trackedResource definition from specification\common-types\resource-management\v2\types.json

Reply:
Done (changed to trackedResource).